2.2.0 changes

cs-abhishek-shukla opened the pull request:
- Rewrote the release notes to add a known issue and workaround
- Edited usage.md to add correlations steps
- Edited contents and other files to add navigation
- Edited the doc for typo grammar and information flow issues

cs-ankit-gupta commented on docs/usage.md:
```diff
@@ -38,4 +38,12 @@ With MITRE ATT&CK&reg; Enrichment Framework, you can configure and schedule data
+    - Under **Correlations** check correlated *Tactics*, *Technique*, *sub-techniques*, and other information
+        - For SOAR Framework `v2.1.1` and earlier, check the *Mitigations* tab
+        - For SOAR Framework `v2.2.0` and later, check the *Recommended ATT&CK Mitigations* tab
+
```
- For SOAR Framework `v2.1.1` and earlier, check *Mitigations* under **Correlations** tab
- For SOAR Framework `v2.2.0` and later, check *Mitigations* under *Recommended ATT&CK Mitigations* tab

cs-ankit-gupta commented on release_notes.md:
```diff
@@ -1,18 +1,57 @@
+
+    > **NOTE**: After an upgrade existing permissions in the **Full App Permission** role remain unaffected.
+
+## Known Issues
```
This need to be added in usage.md. As it is not the part of this release. It would be affected in upcoming version as well.

cs-abhishek-shukla committed:
2.2.0 changes
- Rewrote the release notes to add a known issue and workaround
- Edited usage.md to add correlations steps
- Edited contents and other files to add navigation
- Edited the doc for typo grammar and information flow issues
- Incorporated feedback
